### PR TITLE
Activity Selection in python

### PR DESCRIPTION
**Pull request by shahina-bano**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

I have added the solution code for activity selection in python

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Issue #5114

### CODE REVIEW

**Comment by shahina-bano** [comment]
@reviewers, I am facing an issue in deepsource python since it's showing errors of others code.  So please help out.

File: Python/cp/activity_selection.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    print(activity_selected)
+
+
+if __name__ == "__main__":
+    final_activities()
+
+
+'''
+Test Cases:
+1.
```
Comment (ankitaggarwal23) [inline Python/cp/activity_selection.py]: Remove this and rewrite it as `Sample Output:`

File: Python/cp/activity_selection.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+Input:
+4
+1
+2
+2
+4
+3
+5
+4
```
Comment (ankitaggarwal23) [inline Python/cp/activity_selection.py]: Use proper display message.

File: Python/cp/activity_selection.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+2
+3
+3
+4
+4
+6
+
+Output:
+Enter elements of matrix:
+Following activities are selected :
```
Comment (ankitaggarwal23) [inline Python/cp/activity_selection.py]: Remove the extra test case

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by shahina-bano** [comment]
changes are done.

File: Python/cp/activity_selection.py [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+elements of row 0
+1
+2
+elements of row 1
+2
+4
+elements of row 2
+3
+5
+elements of row 3
```
Comment (ankitaggarwal23) [inline Python/cp/activity_selection.py]: Capitalize letter `E`

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Rest looks good to merge

**Comment by shahina-bano** [comment]
oops, sry for that capital thing, done now :) @ankitaggarwal23

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**